t/13-osutils.t

```diff
@@ -130,4 +130,15 @@ subtest find_bin => sub {
 
 };
 
+subtest runcmd => sub {
+    use osutils 'runcmd';
+
+    my @cmd = ('qemu-img', 'create', '-f', 'qcow2', 'image.qcow2', '30G');
```

Comment:
30G is kind of a big number, even if it takes very short time :P can you switch it to 1G?

Reply:
Fine. I just thought the initial size of qcow2 format is quite small